config/betterquesting/DefaultQuests/Quests/Tier1LV-AAAAAAAAAAAAAAAAAAAAAw==/ANewAlloyMagnali-AAAAAAAAAAAAAAAAAAAEfQ==.json

```diff
@@ -31,7 +31,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "Wondering why you need magnalium now? It\u0027s not for GT machines but for a material update for your crossbow.\n\nGo to the Twilight Forest and find an Olivine/Glauconite vein located at Y 10-40 or in the Overworld a Glauconite Soapstone mix located at Y 20-50 and a Granitic Mineral Vein in the Overworld Located at Y 50-60 (The Gypsum vein you will need for the Bricked Blast Furnace).\n\nWith your Ore Washer you can also get Magnesium out of ores like Magnesite.\n\nYou already made some Aluminium dust before. It will get easier once you have an EBF or an MV electrolyzer.\n\nCombine both in your alloy smelter to get magnalium."
```

Comment:
why did you remove granitic mineral vein and magnesite?

reminding people of previous alu dusts quests was a good thing too, they usually forget.

please overhaul or revert

Reply:
I've put back the reminder about Aluminium in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426/commits/c30e6cb9f8dc370e091305c7963ffe0ad807de25

Just in case, duplicating the reasoning behind the rest from Discord:
If I understood correctly, under "granitic mineral vein" we imply the Mineralsand Vein that contains:
* Basaltic Mineral Sand, Granitic Mineral Sand, Fullers Earth - require MV eletrolyzer to get magnesium
* Gypsum Ore - doesn't contain magnesium

The note about Magnesite was removed because even before the change, the quest asked to get Olivine and other ores that require a player to understand the processing chart, so the note about Magnesite was considered unnecessary because if a player understands how to get Magnesium from Olivine, they sure understand how to get it from Magnesite.